Library/Homebrew/cmd/edit.rb

```diff
@@ -26,13 +26,17 @@ def edit
     else
       # Don't use ARGV.formulae as that will throw if the file doesn't parse
       paths = ARGV.named.map do |name|
-        name = Formulary.canonical_name(name)
-        Formula.path(name)
-      end
-      unless ARGV.force?
-        paths.each do |path|
-          raise FormulaUnavailableError, path.basename('.rb').to_s unless path.file?
+        formula = Formulary.loader_for(name)
```

Comment:
I don't want to start using loader_for like this, it shouldn't be used outside of `formulary.rb`.